docs/astro/src/content/docs/guide/platforms/web.mdx

```diff
@@ -0,0 +1,101 @@
+---
+<!-- Copyright © SixtyFPS GmbH <info@slint.dev> ; SPDX-License-Identifier: MIT -->
+title: Web
+description: Running Slint on the Web with Wasm
+---
+
+While Slint is mainly designed for native applications (desktop, embedded, mobile),
+it can also run in a web browser using WebAssembly (Wasm).
+
+:::caution[Caution]
+Currently, only the Rust language is supported when targeting the web, and the only renderer available is FemtoVG.
+:::
```

Comment:
Here's a suggestion to phrase it more actively, instead of a passive observation of Slint:

(See section 6 of the style guide)

```suggestion
Slint applications run on desktop, embedded, and mobile platforms. Slint applications written with Rust can
also be cross-compiled to WebAssembly (Wasm) and run in the web browser.

:::note[Note]
Slint cross-compiled for Wasm uses the  <Link type="WinitBackend" label="Winit backend" /> with the FemtoVG renderer.
:::
```

Reply:
The point of the warning was to point out that rust is the only language.

Reply:
Instead of that being a warning, I propose to make it part of the regular intro paragraph (second sentence). Caution in markdown notes are used to warn about potential negative consequences. That doesn't make sense here, I think.

Reply:
this just match what is in the tutorial.
I think it's not fair for the C++/JS people reading the docs if this doesn't emphasis that this will not work for them.